documentation/book/ref-tls-sidecar.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-tls-sidecar.adoc
+
+[id='ref-tls-sidecar-{context}']
+= TLS sidecar configuration
+
+TLS sidecar can be configured using the `tlsSidecar` property in:
+
+* `Kafka.spec.kafka`
+* `Kafka.spec.zookeeper`
+* `Kafka.spec.entityOperator`
+
+The TLS sidecar supports two additional options:
+
+* `image`
+* `resources`
+
+The `resources` property can be used to specify the xref:assembly-resource-limits-and-requests-{context}[memory and cpu resources] allocated for the TLS sidecar.
+
+The `image` property can be used to configure the container image which will be used.
+For more details about configuring custom container images, see xref:assembly-configuring-container-images-{context}[].
```

Comment:
replace {context} with str

Reply:
This is not in the `str` context. The assembly is included in the deployment contexts and using the `{context}` variable will make sure that the link will lead to the `assembly-configuring-container-images` in the same context.

Reply:
Okay, you probably want to spell that out then instead of using the attribute.  We have to remember we might move it to another assembly at some point.

Reply:
I guess this is an interesting question. What we have is the following structure:

```
- assembly-X (context=ctx-x)
    - assembly 1
    - assembly 2
- assembly-y (context=ctx-y)
    - assembly 1
    - assembly 2
```

By using the `{context}` in the link we will make sure that the link always remains within the `assembly-X` or `assembly-Y`. Which is good. But it is not stable. Do we want to raise it as another issue? :-)

Reply:
We might have to ... and I hate context right now.  If the link isn't broken as it stands, then let's leave it alone for now.

Reply:
I opened https://github.com/redhat-documentation/modular-docs/issues/72